### PR TITLE
Created initial readme version

### PR DESCRIPTION
**Pull request by dominguesgm**
Created the initial README version.

Based on the readme in [discussify/styleguide](https://github.com/ipfs-shipyard/discussify-styleguide/blob/master/README.md).

Some sections (such as install and setup) were not included as they do not make sense at such an early stage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
-# idm-styleguide
-IDM's living styleguide
+# idm-web-uikit
+
+IDM's living styleguide.
```
Comment (PedroMiguelSS) [inline README.md]: Change this line to: `IDM's living web UI kit` to match `package.json` description field.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+Starts [Storybook](https://storybook.js.org/).
+
+### build
+
+```sh
+$ npm run build
+```
+
+Builds the project.
+
```
Comment (PedroMiguelSS) [inline README.md:31]: Add a note here stating that we'll add this script later.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+Releases and publishes the package. Runs tests, lints and builds the project beforehand.
+
+This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
+
+NOTE: this package still does not have a release script, as it is still in the initial stages of its development. This script will be added in the near future.
+
+
+## Contributing
+
+If you want to contribute for the project, we encourage you to read over the [discussify](https://github.com/ipfs-shipyard/pm-idm) repository README.
```
Comment (PedroMiguelSS) [inline README.md]: Change `discussify` to `IDM`.

**Review by PedroMiguelSS — CHANGES_REQUESTED** [review]
LGTM. Just some minor changes.

**Review by PedroMiguelSS — APPROVED** [review]

**Comment by PedroMiguelSS** [comment]
@satazor, some sections are missing but we will add them later when it makes sense. Do you see something important that we are missing here?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,110 @@
...
+
+Releases and publishes the package. Runs tests, lints and builds the project beforehand.
+
+This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
+
+NOTE: this package still does not have a release script, as it is still in the initial stages of its development. This script will be added in the near future.
+
+## Using a linked version of idm-web-uikit
+
+In some cases, you may want to make changes to IDM's web-uikit at the same time as you work on your project which uses the web-uikit. In order to use a local version of [idm-web-uikit](https://github.com/ipfs-shipyard/idm-web-uikit) and have any web-uikit modifications be reflected live on your project, some pages have to be made in your main project.
```
Comment (satazor) [inline README.md]: Perhaps we should add the following: Some of the instructions below assume you are using Webpack in your main project.

Comment (satazor) [reply README.md]: We should do the same for the discussify style guide as well.

**Review by satazor — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (PedroMiguelSS) [reply README.md]: Ok. It makes sense. Is the rest ok for you?

Comment (satazor) [reply README.md]: Yep!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,110 @@
...
+
+```sh
+$ npm run release
+```
+
+Releases and publishes the package. Runs tests, lints and builds the project beforehand.
+
+This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
+
+NOTE: this package still does not have a release script, as it is still in the initial stages of its development. This script will be added in the near future.
```
Comment (satazor) [inline README.md]: One question: why are we saying this? It seems the tooling is there and the `release` script exists.

Comment (satazor) [reply README.md]: Also in regards to this, please apply the same changes of this PR in this repo: https://github.com/ipfs-shipyard/discussify-styleguide/pull/11

Comment (PedroMiguelSS) [reply README.md]: Ok, @DominguesGM is taking care of the changes.

Comment (dominguesgm) [reply README.md]: You're right, it doesn't make sense to say this as we do in fact have a release script. This paragraph will be removed.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,110 @@
...
+
+Checks if the project has any linting errors.
+
+### release
+
+```sh
+$ npm run release
+```
+
+Releases the package. Runs tests, lints and builds the project beforehand.
```
Comment (satazor) [inline README.md]: I think you missed this:

 ```
If successful, you may publish the release to npm by running `$ npm publish`.
```

**Review by satazor — APPROVED** [review]

**Pull request merged**